docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -192,6 +192,8 @@ This can also be set in the configuration file using
 <<synthetics-configuration-monitor,`monitor.locations` in the configuration file>>.
 The value defined via the CLI will take precedence.
 
+// If Elastic managed locations is disabled, will they see an error message? What will it say?
```

Comment:
Yeah it will throw an error if user will try to use public locations in project monitors 

![image](https://github.com/elastic/observability-docs/assets/3505601/253cdcbc-e687-4ce1-8465-972464927397)